src/sections/quiz/quiz-body.js

```diff
@@ -3,35 +3,119 @@ import {SvgIcon} from "@mui/material";
 import Container from "@mui/material/Container";
 import Box from "@mui/material/Box";
 import Logo from "../../components/svg-color/Logo.svg";
-import {Lora} from '@next/font/google';
-import Grid from '@mui/material/Grid';
+import {Lora} from "@next/font/google";
+import Grid from "@mui/material/Grid";
+import useSWR from "swr"
```

Comment:
I double-checked [Next](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-client-with-route-handlers) docs, they recommend 2 libs: SWR and react query. Since SWR does not support post requests, we will use react query. It supports both [GET](https://tanstack.com/query/latest/docs/react/guides/queries) and [POST](https://tanstack.com/query/latest/docs/react/guides/mutations) requests.